korma/src/commonTest/kotlin/com/soywiz/korma/geom/Matrix3DTest.kt

```diff
@@ -100,32 +99,32 @@ class Matrix3DTest {
             val mat = Matrix(2, 0, 0, 2, 20, 20)
             val mat4 = mat.toMatrix3D()
             assertEquals(Point(40, 40), mat.transform(Point(10, 10)))
-            assertEquals(Vector3D(40, 40, 0), mat4.transform(Vector3D(10f, 10f, 0f)))
+            assertEquals(Vector3D(40f, 40f, 0f), mat4.transform(Vector3D(10f, 10f, 0f)))
```

Comment:
Didn't notice, the constructor receiving Ints was removed?

Hopefully at some point this won't be necessary if Kotlin implements something like this:
https://youtrack.jetbrains.com/issue/KT-39136